src/convertToObject.js

```diff
@@ -2,17 +2,28 @@
 
 /**
  * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .split(';')
+    .filter(row => row.trim().length > 1)
+    .map(element => {
+      styles[element.split(':')[0].trim()]
+        = element.split(':')[1].trim();
```

Comment:
would be great to create variables for `element.split(':')[0].trim()` and `element.split(':')[1].trim()` for more readable code